lession7/exercise2/bt3/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>excercise 2</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://use.fontawesome.com/874f4d88ca.js"></script>
+    <!-- Latest compiled and minified CSS & JS -->
+    <link rel="stylesheet" media="screen" href="//netdna.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
+    <script src="//code.jquery.com/jquery.js"></script>
+    <script src="//netdna.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
+    
+</head>
+<body>
+    <div class="above"></div>
+    <div class="main">
+        <div class="header">
+            <div class="container">
+
+                <div class="tittle">
+                    
+                    <div class="nav">
+                        <a class="home" href="#">Home</a>
+                        <a class="bedRoom" href="#">Bed Room</a>
+                    </div>
+
+                    <div class="menu">
+                        <button>
+                            MENU
+                            <span><i class="fa fa-bars" aria-hidden="true"></i></span>
+                        </button>
+                    </div>
+
+                </div>
+    
+                <div class="intro">
+                    <div class="introText">
+                        <div class="introTextOver"></div>
+                        <div class="introTextBelow"></div>
+                    </div>
+                    <div class="introImg"></div>
+                </div>
+    
+                <div class="service">
+    
+                </div>
+    
+                <div class="slide">
+                    
+                </div>
```

Comment:
Nội dung quá ngắn hãy viết trên 1 dòng
```suggestion
                <div class="slide"></div>
```